src/planner/logical_operator.cpp

```diff
@@ -57,6 +57,7 @@ InsertionOrderPreservingMap<string> LogicalOperator::ParamsToString() const {
 		expressions_info += expressions[i]->GetName();
 	}
 	result["Expressions"] = expressions_info;
+	result["Estimated Cardinality"] = to_string(estimated_cardinality);
```

Comment:
I've changed this to `__estimated_cardinality__`  in https://github.com/duckdb/duckdb/pull/13476/, perhaps we can introduce a similar method that is introduced there in the `PhysicalOperator` to the `LogicalOperator` (`SetEstimatedCardinality`) - or perhaps this can be handled one layer up instead so this is handled for all logical operators?

Reply:
On that note - perhaps this should check `has_estimated_cardinality` as well?